### PR TITLE
Add freeimage-dev to list of dependencies to ogre

### PR DESCRIPTION
**Pull request by f-fl0**
This PR addresses the following issue:
```
[ WARN] [1615544485.336670000]: OGRE EXCEPTION(5:ItemIdentityException): Can not find codec for 'png' image format.
Supported formats are: dds. in Codec::getCodec at /home/builder/aports/backports/ogre/src/ogre-1.9.1/OgreMain/src/OgreCodec.cpp (line 66)
terminate called after throwing an instance of 'Ogre::ItemIdentityException'
  what():  OGRE EXCEPTION(5:ItemIdentityException): Can not find codec for 'png' image format.
Supported formats are: dds. in Codec::getCodec at /home/builder/aports/backports/ogre/src/ogre-1.9.1/OgreMain/src/OgreCodec.cpp (line 66)
```

In order to visualize image topic in `rviz`, `ogre` should be compiled with `png` support (provided by `freeimage`).

### CODE REVIEW

**Comment by at-wat** [comment]
Build failing on kinetic and melodic.

**Comment by f-fl0** [comment]
> Build failing on kinetic and melodic.

I know. I'm working on it. I need to add backport for `freeimage` in alpine 3.7 and 3.8.

File: backports/freeimage.3.8/ENABLE_ON [hunk 1]
```diff
@@ -0,0 +1 @@
+v3.8
```
Comment (at-wat) [inline backports/freeimage.3.8/ENABLE_ON:1]:
```suggestion
v3.7
v3.8
```
and please change directory name to `freeimage.v3.8`.

Comment (f-fl0) [reply backports/freeimage.3.8/ENABLE_ON:1]: Does it mean that only a `freeimage.v3.8` folder is necessary i.e. should I remove the `freeimage.v3.7`folder?

Comment (f-fl0) [reply backports/freeimage.3.8/ENABLE_ON]: Done in c87975ee5f1bf645c71aa8b0c285461037cafcd4

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**